contributors/devel/sig-architecture/vendor.md

```diff
@@ -0,0 +1,172 @@
+
+Too commonly dependency changes have a ripple effect where something
+else breaks unexpectedly.  The first instinct during issue triage
+is to revert a change.  If the change was made to fix some other
+issue and that issue was not documented, then a revert simply
+continues the ripple by fixing one issue and reintroducing another
+which then needs refixed.  This can needlessly span multiple days
+as CI results bubble in and subsequent patches fix and refix and
+rerefix issues.  This may be avoided if the original modifications
+recorded artifacts of the change rationale.
+
+## Sanity checking
+
+After all of this is done, `git status` should show you what files have been
+modified and added/removed.  Make sure to sanity-check them with `git diff`, and
+to `git add` and `git rm` them, as needed.  It is commonly advised to make one
+`git commit` which includes just the dependencies and `go.mod` and `go.sum` files, and
+another `git commit` that includes changes to Kubernetes code to use (or stop
+using) the new/updated/removed dependency.  These commits can go into a single
+pull request.
+
+Before sending your PR, it's a good idea to sanity check that your
+go.mod, go.sum files and the contents of `vendor/` are ok:
+
+```sh
+hack/verify-vendor.sh
+```
+
+## Reviewing and approving dependency changes
+
+Particular attention to detail should be exercised when reviewing and approving
+PRs that add/remove/update dependencies. Importing a new dependency should bring
+a certain degree of value as there is a maintenance overhead for maintaining
+dependencies into the future.
+
+When importing a new dependency, be sure to keep an eye out for the following:
+- Is the dependency maintained?
+- Does the dependency bring value to the project? Could this be done without
+  adding a new dependency?
+- Is the target dependency the original source, or a fork?
+- Is there already a dependency in the project that does something similar?
+- Does the dependency have a license that is compatible with the Kubernetes
+  project?
+
+Additionally:
+- Look at the `go.mod` changes in `k8s.io/kubernetes`.
+  Check that the only changes are what the PR claims them to be. 
+- Look at the `go.mod` changes in the staging components.
+  If they are adding new `replace` directives, this is problematic for
+  consumers of those libraries (it means we are pinned to older versions than
+  would be selected by go when our module is used as a library).
+- Check if there is a tagged release we can vendor instead of a random hash
+- Scan the imported code for things like init() functions
+- Look at the Kubernetes code changes and make sure they are appropriate
+  (e.g. renaming imports or similar). You do not need to do feature code review.
+- If this is all good, approve, but don't LGTM, unless you also do code review
+  or unless it is trivial (e.g. moving from k/k/pkg/utils -> k/utils).
+
+All new dependency licenses should be reviewed by either Tim Hockin (@thockin)
```

Comment:
Isn't there a group alias for this?

Reply:
@kubernetes/dep-approvers should be used for this.

Reply:
we should fix the OWNERS instead of @ing the whole alias always. @ing subscribes all members and should be a fallback or for when the whole group should be looking.